tests/distributed/test_model_parallelization.py

```diff
@@ -295,9 +342,45 @@ def test_model_parallel_without_parallelizing_embeddings(
             from_config=False,
             with_lazy_load=True,
             parallelize_embeddings=False,
+            sequence_parallel_enabled=True,
             overwrite_model_config=config_overwrite,
         )
 
+    # TODO: enable that once ParallelCrossEntropy works.
```

Comment:
Instead of commenting these tests, you could just have skipped them unconditionally.